hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/sps/TestExternalStoragePolicySatisfier.java

```diff
@@ -482,6 +492,8 @@ private void doTestWhenStoragePolicySetToCOLD() throws Exception {
 
     hdfsCluster.triggerHeartbeats();
     dfs.satisfyStoragePolicy(new Path(FILE));
+    // Assert metrics.
+    assertEquals(1, hdfsCluster.getNamesystem().getPendingSPSPaths());
```

Comment:
it is still there, you added a new test as well to get rid of this only right?

Reply:
Thanks for your comments. I'm sorry I left that out.